.storybook/main.js

```diff
@@ -0,0 +1,20 @@
+const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin')
+
+module.exports = {
+  stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
```

Comment:
we probably don't need to support all those different build pipelines so in end we only want to support stories written in typescript files. Which then also might allow us to ditch all those new `@babel` packages as in theory not needed as we use typescript for transpiling/compiling in end so no need for Babel.

Reply:
removed unnecessary file formats 👍🏽